src/main/java/com/pinterest/secor/common/monitoring/OstrichAdminService.java

```diff
@@ -58,7 +57,7 @@ public void start() {
             List$.MODULE$.<StatsFactory>empty(),
             Option.<String>empty(),
             List$.MODULE$.<Regex>empty(),
-            Map$.MODULE$.<String, CustomHttpHandler>empty(),
+            new Map.Map1<>("/prometheus", new PrometheusHandler()),
```

Comment:
Should this line be conditioned on whether prometheus is enabled?

Reply:
Handler will return 404, if there is no PrometheusMetricRegistry added on Micrometer (see handler class itself).
But I guess we could add a check here too, to make it more obvious while reading the code.